src/Microsoft.Health.Dicom.Client/IDicomWebClient.Update.cs

```diff
@@ -0,0 +1,16 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using FellowOakDicom;
+
+namespace Microsoft.Health.Dicom.Client;
+
+public partial interface IDicomWebClient
+{
+    Task<DicomWebResponse> UpdateStudyAsync(IReadOnlyList<string> studyInstanceUids, DicomDataset dataset, int partitionKey = default, CancellationToken cancellationToken = default);
```

Comment:
This needs to be `partitionName`. Users usually dont know what partitionKey is

Reply:
Updated to `partitionName`